packages/SwingSet/test/virtualObjects/test-representatives.js

```diff
@@ -376,10 +384,40 @@ test('virtual object gc', async t => {
     remainingVOs[key] = hostStorage.kvStore.get(key);
   }
   t.deepEqual(remainingVOs, {
-    'v1.vs.vom.es.o+1/3': '1',
+    'v1.vs.vom.es.o+1/3': 'r',
     'v1.vs.vom.o+1/2': '{"label":{"body":"\\"thing #2\\"","slots":[]}}',
     'v1.vs.vom.o+1/3': '{"label":{"body":"\\"thing #3\\"","slots":[]}}',
     'v1.vs.vom.o+1/8': '{"label":{"body":"\\"thing #8\\"","slots":[]}}',
     'v1.vs.vom.o+1/9': '{"label":{"body":"\\"thing #9\\"","slots":[]}}',
   });
 });
+
+test('empty facets are not orphaned', async t => {
+  const config = {
```

Comment:
let's add a comment, "facets which don't reference their `state` should still keep their cohort alive", that might not be obvious to us six months from now